Makefile

```diff
@@ -119,8 +119,9 @@ testraceslow: testslow
 # given all tests in the package will be run).
 .PHONY: stress
 stress:
-	$(GO) test -v $(GOFLAGS) -tags '$(TAGS)' -i -c $(PKG) -o $(PKG)/stress.test
-	cd $(PKG) && stress $(STRESSFLAGS) ./stress.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
+	$(GO) list -tags '$(TAGS)' -f \
+	'$(GO) test -v $(GOFLAGS) -tags '\''$(TAGS)'\'' -ldflags '\''$(LDFLAGS)'\'' -i -c {{.ImportPath}} -o {{.Dir}}/stress.test && (cd {{.Dir}} && stress $(STRESSFLAGS) ./stress.test -test.run $(TESTS) -test.timeout $(TESTTIMEOUT) $(TESTFLAGS))' $(PKG) | \
```

Comment:
Add a line of commentary here about why the magical incantation is better to dissuade others from simplifying.